src/Microsoft.AspNetCore.Hosting/Internal/WebHost.cs

```diff
@@ -99,6 +99,8 @@ public void Initialize()
 
         public virtual void Start()
         {
+            HostingEventSource.Log.HostStart();
```

Comment:
Discussed offline. Keeping this line here and instead moved `_logger.Starting()` up